DER/SecurityBasedDerivatives/SecurityBasedDerivatives.rdf

```diff
@@ -140,7 +94,7 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>debt instrument derivative</rdfs:label>
-		<skos:definition>security-based derivative whose underlier is an individual tradable debt instrument, a credit index, or a custom basket of debt-related assets</skos:definition>
+		<skos:definition>security-based derivative whose underlier is an individual tradable debt instrument, credit index, or custom basket of debt-related assets</skos:definition>
```

Comment:
Bad practice to redundantly enumerate here the subclasses of DebtObservable. And it's inconsistent already (e.g. the text misses subclass BasketOfCreditRisks).
BTW whether a debt instrument is an "asset" is relative - to the party obliged to pay back the debt it's a liability.

Reply:
granted it needs work - again it's an improvement, and we really were looking for incremental progess, since so many files have changed